src/main/java/org/terasology/Lakes/LakeProvider.java

```diff
@@ -68,35 +82,35 @@ public void process(GeneratingRegion region) {
         region.setRegionFacet(LakeFacet.class, lakes);
     }
 
-    protected boolean checkGradient(Vector3i pos, BaseFieldFacet2D facet){
+    protected boolean checkGradient(Vector3i pos, BaseFieldFacet2D facet) {
 
-        Rect2i BB = Rect2i.createFromMinAndMax(pos.x()-3,pos.z()-3,pos.x()+3,pos.z()+3);
+        Rect2i BB = Rect2i.createFromMinAndMax(pos.x() - 3, pos.z() - 3, pos.x() + 3, pos.z() + 3);
```

Comment:
Variables and fields should always start with a lower case character. In this, case I would even got for the "long" name for clarity.

```suggestion
        Rect2i boundingBox = Rect2i.createFromMinAndMax(pos.x() - 3, pos.z() - 3, pos.x() + 3, pos.z() + 3);
```

Reply:
actually thanks, i was unclear to what this expanded to